martchellop/Entretenibit
# Redirect after unsubscribing

After clicking the unsubscribe link, now the user is redirected to the Entretenibit website, and a Sweet Alert is shown with a suitable message.

Testing
---

1. Run `docker-compose up`.

    1. This should start the server and the e-mail sender application.

2. Wait until the e-mails are sent.

3. Access http://www.mailinator.com/ and check the inbox for any currently registered e-mail, such as entretenibit4@mailinator.com.

    1. The current registered e-mails have the form entretenibit#@mailinator.com, where # is a number. 

4. Check if the events e-mail arrived in the mailinator inbox you are checking.

5. Click the "unsubscribe" link in the bottom of the e-mail

6. Confirm that you've been redirected to the Entretenibit website

7. Confirm that a suitable Alert has been shown, with either a failure or success message.

8. Confirm in the database if the user has been unregistered.

9. Try to tamper with the link, by changing the e-mail or the hash. It should bring you to a failure page.
